TwilioVerifySDK/TwilioVerify/Sources/Domain/Manager/TwilioVerify.swift

```diff
@@ -30,6 +30,23 @@ public typealias FactorListSuccessBlock = ([Factor]) -> ()
 ///:nodoc:
 public typealias EmptySuccessBlock = () -> ()
 
+/// Wrapper for [kSecAttrAccessible](https://developer.apple.com/documentation/security/ksecattraccessible)
+public enum KeyAttrAccessible {
+  /// Wrapper of [kSecAttrAccessibleAfterFirstUnlock](https://developer.apple.com/documentation/security/ksecattraccessibleafterfirstunlock)
+  case afterFirstUnlock
+  /// Wrapper of [kSecAttrAccessibleAfterFirstUnlockThisDeviceOnly](https://developer.apple.com/documentation/security/ksecattraccessibleafterfirstunlockthisdeviceonly)
+  case afterFirstUnlockThisDeviceOnly
+
+  var value: CFString {
+    switch self {
+      case .afterFirstUnlock:
+        return kSecAttrAccessibleAfterFirstUnlock
+      case .afterFirstUnlockThisDeviceOnly:
+        return kSecAttrAccessibleAfterFirstUnlockThisDeviceOnly
+    }
+  }
+}
+
 ///Describes the available operations to proccess Factors and Challenges
```

Comment:
<table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/67a3c51041ad75975518f7cce5904b3fc20c28e6/TwilioVerifySDK/TwilioVerify/Sources/Domain/Manager/TwilioVerify.swift#L50"></span>Prefer at least one space after slashes for comments.<br />
<code>comment_spacing</code> <code>TwilioVerify.swift:50</code></td>
    </tr>
  </tbody>
</table>